docs/pages/store/tables.mdx

```diff
@@ -57,6 +61,9 @@ ResourceId offchainTableId = ResourceIdLib.encode({
   typeId: RESOURCE_OFFCHAIN_TABLE,
   name: "Balance"
 });
+
+console.log("Onchain table id:%x", uint256(ResourceId.unwrap(tableId)));
+console.log("Offchain table id:%x", uint256(ResourceId.unwrap(offchainTableId)));
```

Comment:
is this leftover?

Reply:
No, it's something I thought would be useful for people so I left it. I'll make that clearer.